virttest/remote.py

```diff
@@ -513,7 +514,8 @@ class RemoteRunner(object):
     def __init__(self, client="ssh", host=None, port="22", username="root",
                  password=None, prompt=r"[\#\$]\s*$", linesep="\n",
                  log_filename=None, timeout=240, internal_timeout=10,
-                 session=None, preferred_authenticaton='password'):
+                 session=None, preferred_authenticaton='password',
+                 log_function=None):
```

Comment:
This is actually another good step to make the functionality available for the `RemoteRunner` too, very good!